client/build.gradle

```diff
@@ -1,3 +1,57 @@
+plugins {
```

Comment:
btw can you add into manifest version? So that getPackageVersion on class will work?

https://github.com/testcontainers/testcontainers-java/blob/master/modules/mockserver/src/test/java/org/testcontainers/containers/MockServerContainerTest.java#L22

Reply:
out of curiosity - why?

Reply:
to get up in tests same version of the container as the lib, same way as in example

Reply:
I created https://github.com/bsideup/liiklus/issues/159 for this